ReactAndroid/src/main/java/com/facebook/react/devsupport/InspectorPackagerConnection.java

```diff
@@ -281,7 +270,7 @@ public void send(JSONObject object) throws IOException {
         return;
       }
 
-      mWebSocket.sendMessage(RequestBody.create(WebSocket.TEXT, object.toString()));
+      mWebSocket.sendMessage(object.toString());
```

Comment:
This was the manual change. Same as in Upgrade OkHttp to 3.6.0. Based on 93a1d592d61b22 (#17) · airbnb/react-native@6a6c39d